installers/docker-compose/legend-with-remote-gitlab/README.md

```diff
@@ -33,7 +33,7 @@ docker-compose --env-file ~/legend-temp/.env --profile all up --detach
 
```

Comment:
Can you please move the docker/docker-compose installer changes to a different PR ?
They can be merged independent of the other documentation changes in this PR.

Reply:
Updated!